# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gmp/autotools.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Need to correct some files when doing a autogen cycle with newer tools.
	--jsaw

--- gmp-4.1.4/acinclude.m4.orig	2006-03-01 21:28:02.000000000 +0100
+++ gmp-4.1.4/acinclude.m4	2006-03-01 21:28:20.000000000 +0100
@@ -834,7 +834,7 @@
 echo ["define(<CONFIG_TOP_SRCDIR>,<\`$tmp'>)"] >>$gmp_tmpconfigm4
 
 # All CPUs use asm-defs.m4 
-echo ["include(CONFIG_TOP_SRCDIR\`/mpn/asm-defs.m4')"] >>$gmp_tmpconfigm4i
+echo ["include(][CONFIG_TOP_SRCDIR][\`/mpn/asm-defs.m4')"] >>$gmp_tmpconfigm4i
 ])
 
 
--- gmp-4.1.4/mpfr/Makefile.am	2002-05-16 02:27:52.000000000 +0200
+++ gmp-4.1.4-t2/mpfr/Makefile.am	2006-03-02 00:59:58.000000000 +0100
@@ -28,7 +28,7 @@
 SUBDIRS = tests
 
 INCLUDES = -I$(top_srcdir)
-CFLAGS = @MPFR_CFLAGS@
+AM_CFLAGS = @MPFR_CFLAGS@
 LIBS = @MPFR_LIBS@
 
 if WANT_MPFR
--- gmp-4.1.4/mpfr/tests/Makefile.am	2002-11-24 23:00:42.000000000 +0100
+++ gmp-4.1.4-t2/mpfr/tests/Makefile.am	2006-03-02 01:00:14.000000000 +0100
@@ -22,7 +22,7 @@
 AUTOMAKE_OPTIONS = gnu no-dependencies $(top_builddir)/ansi2knr
 
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/mpfr
-CFLAGS = @MPFR_CFLAGS@
+AM_CFLAGS = @MPFR_CFLAGS@
 LIBS = @MPFR_LIBS@
 
 LDADD = ../libmpfr.a $(top_builddir)/libgmp.la
